numpy/random/tests/test_extending.py

```diff
@@ -25,27 +27,29 @@
 except ImportError:
     cython = None
 
+cython_ver = cython.__version__.split('.')
```

Comment:
This should go within an `else` of the try/except above, else will raise AttributeError.